app/web/src/newhotness/Onboarding.vue

```diff
@@ -608,41 +608,64 @@ export const DEBUG_PROVIDER_CHOICE = undefined;
                       </div>
                       <div>
                         <span>
-                          Clone the
-                          <NewButton
-                            aria-label="ai-agent-repo-main"
+                          Install the SI CLI locally
+                          <!-- <NewButton
+                           aria-label="si-cli-link"
+                           :class="
+                             clsx(
+                               'underline',
+                               themeClasses(
+                                 'hover:text-action-500',
+                                 'hover:text-action-300',
+                               ),
+                             )
+                           "
+                           tone="nostyle"
+                           href="" <----- we need to link our to where we talk about the CLI
+                           target="_blank"
+                           label="setup script"
+                           @mousedown="
+                             onboardingTracking('external_link_si_cli')
+                           "
+                         /> -->
+                        </span>
+                        <CopyableTextBlock
+                          :text="installCommand"
+                          @copied="
+                            onboardingTracking(
+                              'copied_si_cli_install_instructions',
+                            )
+                          "
+                        />
+                        <span
+                          ><NewButton
+                            aria-label="si-cli-install-link"
                             :class="
                               clsx(
-                                'underline',
+                                'hover:underline',
                                 themeClasses(
-                                  'hover:text-action-500',
-                                  'hover:text-action-300',
+                                  'text-neutral-800',
+                                  'text-neutral-300',
                                 ),
                               )
                             "
                             tone="nostyle"
-                            href="https://github.com/systeminit/si-ai-agent"
+                            href="https://docs.systeminit.com/reference/si-cli#installation"
                             target="_blank"
-                            label="AI Agent repository"
+                            label="[Full installation instructions]"
```

Comment:
```suggestion
                            label="Alternative setup options"
```

Reply:
The "full installation instructions" feels like something to check or follow. Saying "alternative" and "options" sound more optional.